src/azul/service/drs_controller.py

```diff
@@ -300,17 +304,27 @@ def encode_access_id(token_str: str, replica: str) -> str:
 
     >>> decode_access_id(encode_access_id('back on boogie street', 'aws'))
     ('back on boogie street', 'aws')
+
+    >>> literal = repr(({'not a': 'string'}, 'aws')).encode()
```

Comment:
I'm a little confused for why this doc_test is so long, are you trying to showcase someone encoding their own access ID and (eventually) passing it into the system?

Otherwise 
```
decode_access_id(encode_access_id('hi jesse',2333))
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "/Users/amar/dev/hca/azul/src/azul/service/drs_controller.py", line 326, in decode_access_id
    raise ValueError('Malformed access ID')
ValueError: Malformed access ID
```
Could also work.

Reply:
I tried that originally, but the type hinting for `encode_access_id()` caused a warning. Suppressing the warning is difficult and obtuse because this is within a docstring. The PyCharm comment showed up before the beginning of the triple quotes which was odd and unintuitive since I only wanted to suppress on line.

I've simplified the doctest though. Let me know what you think.